CMakeLists.txt

```diff
@@ -84,7 +84,13 @@ if(${CUDA_VERSION} VERSION_GREATER_EQUAL "11.0")
   if(NOT (${ARCH} MATCHES "aarch64"))
     cmake_dependent_option(BUILD_NVJPEG2K "Build with nvJPEG2K support" ON
                            "NOT BUILD_DALI_NODEPS" OFF)
+  else()
```

Comment:
you can merge the two conditons, to avoid the double else. Somethin like (please check that it works):
```
if(${CUDA_VERSION} VERSION_GREATER_EQUAL "11.0" AND NOT ${ARCH} MATCHES "aarch64" )
```

Reply:
Done